C/Cpp_include/c4ConnectedClient.hh

```diff
@@ -26,6 +26,22 @@ struct C4ConnectedClient  : public fleece::RefCounted,
                     
     /// Gets the current revision of a document from the server.
     virtual litecore::actor::Async<C4DocResponse> getDoc(C4Slice, C4Slice, C4Slice, bool) noexcept=0;
+    
+    /// Pushes a new document revision to the server.
+    /// @param docID  The document ID.
+    /// @param collectionID  The name of the document's collection, or `nullslice` for default.
+    /// @param revID The new revision ID
+    /// @param parentRevID The ID of the parent revision on the server,
+    ///                      or `nullslice` if this is a new document.
+    /// @param revisionFlags  Flags of this revision.
+    /// @param fleeceData  The document body encoded as Fleece (without shared keys!)
+    /// @return  An async value that, when resolved, contains the status as a C4Error.
+    virtual litecore::actor::Async<void> updateDoc(C4Slice docID,
```

Comment:
C++ APIs should use `slice` instead of `C4Slice`. (This applies to the getDoc method too; I just didn't notice before.)